ios/Classes/ElementTargeting/FlutterElementTargeting.swift

```diff
@@ -0,0 +1,89 @@
+import Flutter
+import UIKit
+import AppcuesKit
+
+internal class FlutterElementSelector: AppcuesElementSelector {
+    private enum CodingKeys: String, CodingKey {
+        case identifier
+    }
+
+    let identifier: String
+
+    init?(identifier: String?) {
+        // must have at least one identifiable property to be a valid selector
+        guard let identifier = identifier else {
+            return nil
+        }
+
+        self.identifier = identifier
+        super.init()
+    }
+
+    override func evaluateMatch(for target: AppcuesElementSelector) -> Int {
+        guard let target = target as? FlutterElementSelector else {
+            return 0
+        }
+
+        return identifier == target.identifier ? 10_000 : 0
+    }
+
+    override func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        if !identifier.isEmpty {
+            try container.encode(identifier, forKey: .identifier)
+        }
+    }
+}
+
+@available(iOS 13.0, *)
+internal class FlutterElementTargeting: AppcuesElementTargeting {
```

Comment:
the screenshot works out of the box on iOS side, so no custom work here on that aspect.  Similar behaviors though around maintaining the `widgets` collection and using this for layout capture, as well as the selector with `"identifier"` values.